Safety/AttitudeManager/Inc/AttitudeDatatypes.hpp

```diff
@@ -6,25 +6,36 @@
 // Output of the PID module and input to the OutputMixing module
 typedef struct
 {
-    float rollPercent;
-    float pitchPercent;
-    float rudderPercent;
-    float throttlePercent;
+    float frontLeftMotorPercent;
+    float frontRightMotorPercent;
+    float backLeftMotorPercent;
+    float backRightMotorPercent;
 
 } PID_Output_t;
 
-#ifdef SPIKE
-    constexpr static uint8_t L_TAIL_OUT_CHANNEL {0}; // Spike has ruddervators
-    constexpr static uint8_t R_TAIL_OUT_CHANNEL {1};
-#else
-    constexpr static uint8_t ELEVATOR_OUT_CHANNEL {0};
-    constexpr static uint8_t RUDDER_OUT_CHANNEL {1};
-#endif
+constexpr static uint8_t FRONT_LEFT_MOTOR_CHANNEL {0};
+constexpr static uint8_t FRONT_RIGHT_MOTOR_CHANNEL {1};
+constexpr static uint8_t BACK_LEFT_MOTOR_CHANNEL {2};
+constexpr static uint8_t BACK_RIGHT_MOTOR_CHANNEL {3};
```

Comment:
same as above, renaming according to quad

Reply:
Do we want these in here or safetyConfig.hpp

Reply:
Also double checking, this is as if the drone is pointed forwards and our front is the same as the front of the drone and we are looking at it from above?

Reply:
yes